int32 id
---
bool setreset_para_accepted